internal/datastore/postgres/migrations/zz_migration.0012_add_xid_constraints.go

```diff
@@ -0,0 +1,56 @@
+package migrations
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/jackc/pgx/v4"
+)
+
+const (
+	getNSConfigPkeyName = `
```

Comment:
It's because one of the migrations added a constraint without a name:
https://github.com/authzed/spicedb/blob/main/internal/datastore/postgres/migrations/zz_migration.0008_add_ns_config_id.go#L11

I didn't want to assume that it would always follow a naming convention.

Reply:
ah, missed that one 👍🏻